pkg/util/sys/device.go

```diff
@@ -289,23 +289,24 @@ func UnmountDevice(devicePath string, executor exec.Executor) error {
 	return nil
 }
 
-func CheckIfDeviceAvailable(executor exec.Executor, name string) (bool, string, error) {
+func CheckIfDeviceAvailable(executor exec.Executor, name string) (int, bool, string, error) {
```

Comment:
Can we get a godoc comment-header for this function that spells out the return values?